projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -708,6 +712,18 @@ export class IgxTimePickerComponent implements
         this._dialogOverlaySettings = {};
     }
 
+    /**
+     * @hidden
+     */
+    public ngAfterViewInit(): void {
+        if (this.mode === TimePickerInteractionMode.dropdown) {
+            fromEvent(this.input.nativeElement, 'keydown').pipe(
+                throttle(() => interval(0, animationFrameScheduler)),
```

Comment:
There is no sense to throttle events for 0 ms. I think setting the interval to 300 ms would make more sense.

Reply:
**interval(0, animationFrameScheduler)** creates an observable that emits one event every time the browser is ready to display a new frame and then the throttle operator will emit a value from that source observable.